pvt/benchmarks/src/PoolBenchmark.behavior.ts

```diff
@@ -180,6 +183,65 @@ export class Benchmark {
           receipt
         );
       });
+
+      it('measures gas (BatchRouter)', async () => {
+        // Warm up
+        let tx = await batchRouter.connect(alice).swapExactIn(
+          [
+            {
+              tokenIn: poolTokens[0],
+              steps: [
+                {
+                  pool: this.pool,
+                  tokenOut: poolTokens[1],
+                  isBuffer: false,
+                },
+              ],
+              exactAmountIn: SWAP_AMOUNT,
+              minAmountOut: 0,
+            },
+          ],
+          MAX_UINT256,
+          false,
+          '0x'
+        );
+        let receipt = await tx.wait();
+        await saveSnap(
+          this._testDirname,
+          `[${this._poolType} - ${gasTag} - BatchRouter] swap single token exact in with fees - cold slots`,
+          receipt
+        );
+
+        await actionAfterFirstTx();
+
+        // Measure
+        tx = await batchRouter.connect(alice).swapExactOut(
```

Comment:
This one should also be exact in according to the gas file name below.

Reply:
Apparently this error was also there for the regular router; can we fix that one too?